packages/manager/modules/pci/src/projects/project/instances/instances.routing.js

```diff
@@ -54,6 +56,8 @@ export default /* @ngInject */ ($stateProvider) => {
                 ),
             ),
           ),
+      isInstancesStateScope: /* @ngInject */ ($state) => () =>
+        $state.current.name.startsWith(stateName),
```

Comment:
Won't this be always true ? Your state is 'pci.projects.project.instances' so i assume current state will always start with 'pci.projects.project.instances'. Does state.current refers to the previous state ?

Reply:
No is not always true, it's true only if customers stay on the one of instances or instances subs states.
```
true -> pci.projects.project.instances.* (instances is included)
false -> others states   
```

Reply:
If you want to test if the current state is a child of another, you should use `$state.includes`
https://ui-router.github.io/ng1/docs/latest/classes/state.stateservice.html#includes

Reply:
thnaks, I no longer need.
Thanks.